code/datums/outfits/jobs/engineering.dm

```diff
@@ -3,7 +3,6 @@
 	belt = /obj/item/weapon/storage/belt/utility/full
 	shoes = /obj/item/clothing/shoes/workboots
 	pda_slot = slot_l_store
-	back = /obj/item/weapon/storage/backpack/satchel_eng
```

Comment:
Why was this causing duplicate backpacks?

Reply:
So, to my understanding, because backpacks are inherited from somewhere, when you set it to manually give an outfit a backpack, it puts one on their back, one in their hand.

Engineer is the only role I put a backpack on manually and the only role I noticed had the problem.